5_analyze_data/notebooks/Heatmap_analysis/nf1_heatmap.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Create correlation heatmap from normalized and feature selected NF1 data
+
+# ## Import libraries
+
+# In[1]:
+
+
+import matplotlib.pyplot as plt
+import pathlib
+import pandas as pd
+import seaborn as sb
+
+
+# ## Read in NF1 data `csv`
+
+# In[2]:
+
+
+norm_fs_data = pathlib.Path("../../../4_processing_features/data/nf1_sc_norm_fs_cellprofiler.csv.gz")
+
+data = pd.read_csv(norm_fs_data, index_col=0)
+data
+
+
+# ## Helper function to split `csv` into metadata and features
+
+# In[3]:
+
+
+def split_data(pycytominer_output: pd.DataFrame):
```

Comment:
That makes sense! The only difference is what they return, so I will definitely keep this in mind and see what I can do :smile:

Reply:
I think for now I will save this for another PR. I agree that this function can be one function shared and I would just need to fix what it returns.

@d33bs Is there a way to star or mark something in a PR for future use?

Reply:
@jenna-tomkinson - Sounds good! One way to mark this might be through a new issue via the "..." button on these comments (see image).
![image](https://user-images.githubusercontent.com/3738008/213013903-c5af5465-b16a-4192-9ab9-6d452416edd6.png).